itest/loadtest/mint_batch_test.go

```diff
@@ -194,7 +196,18 @@ func mintBatchStressTest(t *testing.T, ctx context.Context,
 			},
 		},
 	)
-	require.NoError(t, err)
+	if err != nil {
+		require.ErrorContains(t, err, "universe server already added")
```

Comment:
Good point! Though we cannot use `ErrorIs` because the error is being sent over RPC and cannot directly be unwrapped/matched. But we can still use the error constant in `ErrorContains`, which I'll fix.